#! /bin/sh /usr/share/dpatch/dpatch-run
## 10-replace-sysconfig-etc_default.dpatch by Mario Izquierdo (mariodebian) <mariodebian@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Copy config file to /etc/default instead of /etc/syconfig

@DPATCH@
diff -urNad p910nd-0.92~/Makefile p910nd-0.92/Makefile
--- p910nd-0.92~/Makefile	2008-01-02 12:39:48.000000000 +0100
+++ p910nd-0.92/Makefile	2009-02-08 17:54:56.000000000 +0100
@@ -23,7 +23,7 @@
 MANPAGE = p910nd.8
 INSTALL = install
 BINDIR = /usr/sbin
-CONFIGDIR = /etc/sysconfig
+CONFIGDIR = /etc/default
 SCRIPTDIR = /etc/init.d
 MANDIR = /usr/share/man/man8
 
